src/components/date-picker/DatePicker.stories.tsx

```diff
@@ -81,10 +81,10 @@ export const withRange = () => {
     const MAX_TIME = new Date('3000-01-01T00:00:00.000Z');
     const MIN_TIME = new Date(0);
     const TODAY = new Date();
-    const componentStore = new Store({
+    const componentStore: Store<{ date: Date; fromDate?: Date; toDate?: Date }> = new Store({
         date: new Date(),
-        fromDate: null,
-        toDate: null,
+        fromDate: undefined,
+        toDate: undefined,
```

Comment:
Will `null` still work here, or do we need to refactor usages?

Reply:
Good catch. Fixed!